utilities/blob_db/blob_db_test.cc

```diff
@@ -35,17 +35,7 @@ class BlobDBTest : public testing::Test {
   void Open(BlobDBOptionsImpl bdb_options = BlobDBOptionsImpl(),
             Options options = Options()) {
     options.create_if_missing = true;
-    Reopen(bdb_options, options);
-  }
-
-  void Reopen(BlobDBOptionsImpl bdb_options = BlobDBOptionsImpl(),
-              Options options = Options()) {
-    if (blob_db_) {
-      delete blob_db_;
-      blob_db_ = nullptr;
-    }
     ASSERT_OK(BlobDB::Open(options, bdb_options, dbname_, &blob_db_));
-    ASSERT_TRUE(blob_db_);
```

Comment:
I spent two hours on this thing. The complain was that blob_db_ can be null but we called some functions. I tried all the way to hack the code to suppress the warning and the only way I made it work is this: not to support close and open (which we don't seem to use for now). By the way, there seems to be some memory leak cases in BlobDB::Open() in failure cases.